src/ReflectionExtension.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2022, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+declare(strict_types=1);
+/**
+ * Parser Reflection API
+ *
+ * @copyright Copyright 2022, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\ParserReflection;
+
+use ReflectionExtension as BaseExtension;
+
+/**
+ * General parser-level reflection extension
+ */
+class ReflectionExtension extends BaseExtension
```

Comment:
We can avoid this class, it is impossible (without `lisachenko/z-engine` of course) to create extension class from raw PHP source code and just from an AST tree :)